### PR TITLE
Code Review Request for the 4th HTML&CSS Project- Building with Backgrounds and Gradients

### PR DESCRIPTION
**Pull request by Prabhakarzx**
### Hi! Greetings of the day!
In this project we're required to create the clone of old apple.com's home page.
This project is for learning:
- Building with Backgrounds and Gradients
- Image integration via CSS
- Positioning Nav bar above header background.
- Adding logo and search box to the navbar.
**We enjoyed working on this project. And learned gradient and its uses.**
_I'll like to point out that, there is mouse hover effect showing in the background of searchbox and we think it should not. So, tried some ways but couldn't do it._
If its okay then fine, otherwise kindly guide through it.
### Thank You So Much! :bouquet: :bouquet:

### CODE REVIEW

**Comment by elmejdki** [comment]
### Pull request  under review by Zakariae

File: README.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+In this task we were required to create a clone of old Apple.com's home page using the best practices of HTML AND CSS with GitHub process flow.
+
+## Built With
+
+- HTML,
+- CSS3,
+
+## Live Demo
+
+[Live Demo Link]()
```
Comment (elmejdki) [inline README.md]: fix the live demo link you can use [githack](https://raw.githack.com/) for generating live demos.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,50 @@
 # PrabhakarAndAlex-Old-Apple.com-Clone
 This project is focused on learning gradient and background image positioning using CSS. In this project we're required to create clone of Old Apple.com's homepage.
+
+![screenshot](images/SiteScreenshot.png)
```
Comment (elmejdki) [inline README.md]: fix the image link it's not showing on the readme file.

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    </h5>
+</div>
+
+</header>
+
+    <div class="lowerContainer">
+        <div class="lowerImg1"></div>
+        <div class="lowerImg2"></div>
+        <div class="lowerImg3"></div>
+        <div class="lowerImg4"></div>
```
Comment (elmejdki) [inline Index.html]: please changes these div's into link tags.

File: CSS/style.css [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+header {
+  background-image: url(../images/hero-image.jpg);
+  background-position: 5px -744px;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 570px;
+  margin-left: auto;
+  margin-right: auto;
+}
```
Comment (elmejdki) [inline CSS/style.css]: The CSS background should cover the whole page irrespective of the screen size. you can see that in big screens the background and the whole page goes to the left you should make it cover the whole page by using background-size: cover;, here what I have:
image

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+  <meta name="description" content="Old Apple.com's Home Page Clone Project Made by Prabhakar and Alex" />
+  <meta name="keywords" content="HTML, CSS, Old Apple.com Home Page Clone Project, The Odin Project" />
+  <meta name="author" content="Prabhakar and Alex" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="CSS/style.css" rel="stylesheet" />
+</head>
+<body>
+
+<header class="addRow">
```
Comment (elmejdki) [inline Index.html]: **[OPTIONAL]** I think you are not having a good semantic HTML structure you need to use:
- header then main then footer, consider using that.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job so far team :clap: :clap:, you have made a really cool page, but in order to make it approved you need to do some changes.
### status: changes required :recycle: 

- [x] please fix all linter errors:
image

### please fix all required changes before you request another code review.
_If you were stuck or you need help with anything don't hesitate to contact me on slack_
### Happy Coding

File: README.md (hunk 1 above)
Comment (Prabhakarzx) [reply README.md]: Adding it. Thank You!

File: README.md (hunk 2 above)
Comment (Prabhakarzx) [reply README.md]: Fixing in upcoming commit! Thank You!

File: Index.html (hunk 3 above)
Comment (Prabhakarzx) [reply Index.html]: All **Div**s are now **Span**s with anchor tags! Thanks!

File: CSS/style.css (hunk 4 above)
Comment (Prabhakarzx) [reply CSS/style.css]: Tried adding the suggested property. I was creating structure mismatch! So with little work around tried fixing. Thanks!

File: Index.html (hunk 5 above)
Comment (Prabhakarzx) [reply Index.html]: Thanks for suggesting! Elements were too few so main was vomited but now added in the lower section. Thanks!

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @Prabhakarzx! 👋 
You have done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There is just one issue that needs to be resolved before you can move on: ⬇️ 
****

- [x] As was pointed out to you by the previous TSE, the background should fill the entire page no matter what the screen size. You have not yet achieved this. 
Project requirements from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/) 👇 
Screen Shot 2020-06-21 at 10 43 35 PM
Your clone 👇 
Screen Shot 2020-06-21 at 10 45 41 PM



****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by Prabhakarzx** [comment]
Added `Background Size:cover` 
I hope it fix the issue. If not, please help!
Thanks!

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
**HELLO** @Prabhakarzx **GOOD PROGRESS SO FAR. HOWEVER, YOUR PROJECT STILL HAVE SOME PROBLEMS KINDLY FIX THE FOLLOWING** :arrow_down: 

- [x] Your project background images are not well-positioned still Kindly use [this article](https://www.w3schools.com/cssref/pr_background-position.asp) to get an idea of how to properly position your background image :smile: 

overflow

- [x] Also kindly check on your footer images. They should fill the entire container. You can use the flexbox to solve that. Check [this article](https://www.w3schools.com/css/css3_flexbox.asp) :smile: 

try using flex

- [x] The image you have used in your readme file does not project the actual image of the project. Kindly check on that
project imgae

- [x] There is a space between the header image and the footer block image. kindly fix on this too :smile: 
space betwwen

Once you are through with all these changes kindly submit your project for another review :smile: 

If you have any questions kindly contact me via slack on @peterrobert

### HAPPY CODING :computer:

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, great job with this project :+1: However, I could not fully review the project because I couldn't scroll to see the footer or the bottom images. As pointed out by the previous TSE's, I needed to access the footer but I couldn't. Kindly check the image below. It shows how the page is and without scroll even with the direction keys
pt
Also, kindly update the README file with the latest live project. I could review with that but it still has the horizontal scroll

Kindly fix this and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Prabhakarzx** [comment]
### Dear @addod19, @elmejdki, @peterrobert, @BrittanyBlake,
**Thank You for adding valuable comments and constructive feedback!**
**The Project is completely ready and redefined now!**
_We hope it get through now!_
Thanks!

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @Prabhakarzx, congratulations :fireworks: :champagne: The project now meets all the minimum requirements :100: 

Go ahead and merge to master, do not forget to submit the project completion form :100: 
Good luck with your next projects
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Prabhakarzx** [comment]
### Thank You So much @addod19 And The TSE Team!
**Once again you people proved that constructive feedback can greatly improve the outcome!**
_Thank You!_

**Pull request merged**